tests/Application/config/routes.yaml

```diff
@@ -1,3 +1,3 @@
-bitbag_sylius_shipping_export_plugin:
```

Comment:
Should empty configuration files be present in the repository?

Reply:
It is not empty. It contains

```
bitbag_shipping_export_plugin:
    resource: "@BitBagSyliusShippingExportPlugin/Resources/config/routing.yml"
    prefix: /admin

```

Reply:
I know this is empty, but there other configuration files in this directory that are simply empty.